dhis-2/dhis-services/dhis-service-core/src/main/java/org/hisp/dhis/program/DefaultProgramStageInstanceService.java

```diff
@@ -310,7 +311,9 @@ public void auditDataValuesChangesAndHandleFileDataValues( Set<EventDataValue> n
         }
         else
         {
-            programStageInstance.setEventDataValues( updatedOrNewDataValues );
+            Set<EventDataValue> nonAccessibleDataValues = programStageInstance.getEventDataValues().stream().filter( dv -> nonAccessibleDataElements.contains( dv.getDataElement() ) ).collect(  Collectors.toSet() );

```

Comment:
minor: can you format to respect the 80 chars line length:

```
Set<EventDataValue> nonAccessibleDataValues = programStageInstance.getEventDataValues().stream()
    .filter( dv -> nonAccessibleDataElements.contains( dv.getDataElement() ) )
    .collect( Collectors.toSet() );
```

Reply:
yes, applied the style.